include/RED4ext/Scripting/Natives/entAnimationControllerComponent.hpp

```diff
@@ -0,0 +1,94 @@
+#pragma once
+
+// clang-format off
+
+// This file is generated from the Game's Reflection data
```

Comment:
Please remove comments that disable format checking and say the file is generated :)